### DISCUSSION TITLE
giscus_astro

### DISCUSSION THREAD

**Discussion by giscus** [TheZalRevolt/TheZalDev_Blog#11] [Announcements]
# giscus_astro

A little tutorial on how to integrate Giscus on your Astro Blog

http://localhost:3000/giscus_astro

**Comment by TheZal** [comment]
![Hello there](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExNTMyMDdmMDEwN2Y1MmU3NWZiM2EzOGE3NDZmNDExMmU5MDkxZjMyZCZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/Nx0rz3jtxtEre/giphy.gif)

**Reply by dydFreeman** [reply]
![General Kenobi](https://media.tenor.com/h8ESfI_EBaQAAAAC/star-wars-general-grievous.gif)

**Comment by Balastrong** [comment]
Cool, thanks :D
